src/main/scala/org/supersrsFP/lulzio/UnsafeHelper.scala

```diff
@@ -0,0 +1,17 @@
+package org.supersrsFP.lulzio
+
+
+import sun.misc.Unsafe
+import java.lang.reflect.Field
+object UnsafeHelper {
+
+  val Unsafe: Unsafe = {
+
+    val f: Field = classOf[Unsafe].getDeclaredField("theUnsafe")
+    f.setAccessible(true)
+    f.get(null).asInstanceOf[Unsafe]
+  }
+
+  val boxOffset: Long =
+    Unsafe.objectFieldOffset(classOf[Box].getDeclaredField("item"))
```

Comment:
MIGHT BE WORTH STORING THIS ON THE `BOX.java` CLASS AS A STATIC FIELD??? ALSO I THINK THIS SHOULD BE A DEF.